Airports/EGHO/SMR/Regions.txt

```diff
@@ -0,0 +1,451 @@
+N051.12.41.204 W001.36.20.914
+N051.12.39.263 W001.36.21.994
+N051.12.39.267 W001.36.22.144
+N051.12.39.362 W001.36.22.393
+N051.12.39.181 W001.36.22.648
+N051.12.39.009 W001.36.22.229
+N051.12.38.874 W001.36.22.229
+N051.12.38.850 W001.36.22.379
+N051.12.38.977 W001.36.22.831
+N051.12.38.788 W001.36.23.020
+N051.12.38.571 W001.36.22.438
+N051.12.38.506 W001.36.22.406
+N051.12.38.391 W001.36.22.484
+N051.12.38.370 W001.36.22.628
+N051.12.38.583 W001.36.23.229
+N051.12.38.411 W001.36.23.445
+N051.12.38.153 W001.36.22.798
+N051.12.38.026 W001.36.22.674
+N051.12.37.928 W001.36.22.706
+N051.12.37.895 W001.36.22.850
+N051.12.38.178 W001.36.23.628
+N051.12.37.977 W001.36.23.877
+N051.12.37.608 W001.36.22.920
+N051.12.37.493 W001.36.22.927
+N051.12.37.456 W001.36.23.188
+N051.12.37.751 W001.36.24.058
+N051.12.37.608 W001.36.24.234
+N051.12.37.144 W001.36.23.106
+N051.12.35.948 W001.36.23.675
+N051.12.35.908 W001.36.23.361
+N051.12.36.940 W001.36.22.760
+N051.12.36.773 W001.36.21.567
+N051.12.37.068 W001.36.21.457
+N051.12.37.210 W001.36.22.503
+N051.12.37.337 W001.36.22.673
+N051.12.37.648 W001.36.22.470
+N051.12.37.513 W001.36.21.306
+N051.12.37.072 W001.36.21.456
+N051.12.35.680 W001.36.21.947
+N051.12.35.516 W001.36.20.954
+N051.12.37.746 W001.36.20.145
+N051.12.37.884 W001.36.20.633
+N051.12.38.144 W001.36.20.737
+N051.12.38.172 W001.36.20.992
+N051.12.37.728 W001.36.21.221
+N051.12.37.913 W001.36.22.248
+N051.12.38.880 W001.36.21.836
+N051.12.38.787 W001.36.21.237
+N051.12.39.010 W001.36.21.159
+N051.12.39.120 W001.36.21.725
+N051.12.39.734 W001.36.21.378
+N051.12.39.653 W001.36.20.908
+N051.12.39.842 W001.36.20.851
+N051.12.39.931 W001.36.21.248
+N051.12.40.591 W001.36.20.927
+N051.12.40.583 W001.36.20.927
+N051.12.40.259 W001.36.20.640
+N051.12.40.038 W001.36.20.528
+N051.12.40.018 W001.36.20.777
+N051.12.38.232 W001.36.21.457
+N051.12.38.044 W001.36.20.018
+N051.12.38.978 W001.36.19.691
+N051.12.39.047 W001.36.20.221
+N051.12.39.363 W001.36.20.103
+N051.12.39.285 W001.36.19.613
+N051.12.39.854 W001.36.19.351
+N051.12.39.953 W001.36.20.077
+N051.12.40.804 W001.36.20.685
+N051.12.41.009 W001.36.20.653
+N051.12.41.296 W001.36.20.247
+N051.12.41.341 W001.36.19.999
+N051.12.41.046 W001.36.17.727
+N051.12.40.960 W001.36.17.577
+N051.12.40.919 W001.36.17.361
+N051.12.40.670 W001.36.15.376
+N051.12.40.417 W001.36.14.187
+N051.12.40.130 W001.36.13.658
+N051.12.39.802 W001.36.13.318
+N051.12.39.278 W001.36.13.200
+N051.12.39.086 W001.36.13.259
+N051.12.38.811 W001.36.14.174
+N051.12.38.778 W001.36.14.357
+N051.12.39.077 W001.36.14.586
+N051.12.40.413 W001.36.14.168
+N051.12.40.670 W001.36.15.376
+N051.12.37.517 W001.36.16.482
+N051.12.35.263 W001.36.17.193
+N051.12.35.111 W001.36.16.217
+N051.12.38.319 W001.36.15.073
+N051.12.38.700 W001.36.13.916
+N051.12.38.774 W001.36.13.432
+N051.12.38.762 W001.36.13.242
+N051.12.38.663 W001.36.13.144
+N051.12.34.369 W001.36.12.923
+;GrassTaxiway
+REGIONNAME Thruxton
+grass N051.12.35.770 W001.35.59.727

```

Comment:
Is this the same colour scheme as the Cambridge SMR? :)

Reply:
Wasn't meant to be haha

Reply:
Fairs, think it might be worth going for a standard design across airfields at some point. But this is good for now!